test/resources/hvdbasic/train_hvd_env_vars.py

```diff
@@ -0,0 +1,19 @@
+import json
+import os
+import horovod.tensorflow as hvd
+
+hvd.init()
+
+with open('/opt/ml/model/local-rank-%s-rank-%s' % (hvd.local_rank(), hvd.rank()), 'w+') as f:
+    basic_info = {'local-rank': hvd.local_rank(), 'rank': hvd.rank(), 'size': hvd.size()}
+
+    print(basic_info)
+    json.dump(basic_info, f)
+
+val = os.environ.get('AWS_CONTAINER_CREDENTIALS_RELATIVE_URI')
+host = os.environ.get('SM_CURRENT_HOST')
+
+assert val is not None
+assert host is not None
+
+print('host {}: AWS_CONTAINER_CREDENTIALS_RELATIVE_URI={}'.format(host, val))
```

Comment:
it might be better to also check that this value is different per node, i.e. write the value to a file, and then check in the test code that the different nodes returned different values?

Reply:
Good call, will follow up with this approach in another PR.